content/definitions/_index.en.md

```diff
@@ -1,9 +1,24 @@
+
+UX was concepted in the US by Don Norman in the early 1990s, and is inclusive of Interaction Design, Content Design, Visual Design, and User Research. Quoting Norman, "I wanted to cover all aspects of the person's experience with the system including industrial design graphics, the interface, the physical interaction and the manual." The roots of UX lie in traditional product design, conceptual design, as well as in Human Computer Interaction (HCI) design. HCI is an academic approach to mostly interaction design, that has its roots in the early days of computing before color monitors or mice. Cognitive science and computer science, are the two disciplines that most directly lead into HCI; with industrial design and graphic design, the two disciplines that most directly lead into UX.
+
+- **Interaction Design** categorically speaks to how tasks are broken-up across a multitude of screens, states, and actions. An experience can have terrific interaction design, yet still lack a consistent visual grid, clear hierarchies, or actionable items "visible" to vision impaired users. 
+- **Visual Design** speaks to aesthetic attributes that contribute to shaping digital experiences and their usability (and yes, is typically limited to visual or time-based aesthetics). Branding is a wholly separate design dicipline, however usually falls on the shoulders of a visual designer, when project resources are limited. 
+- **Content Design** speaks to the voice and tone of written language, illustrations, photos, and infographics. 
+- **Accessibility Design** is considering how interfaces are experienced by folks with hearing, visual, motor skills, or other impairments. Folks who may or may not use assistive devices such as screen readers or bone conducting headsets, or who may just fully navigate UIs with a keyboard. 
+- **Participatory Design** [or "Co-Design"](https://en.wikipedia.org/wiki/Participatory_design) is an approach to shaping experiences in collaboration with end-users. Practicioners of participatory design function much more as "moderators" of collaborative sessions, than as "creators" of artifacts.
+- **User Research** is the discipline of learning from users. This could be done in a multitude of ways. User interviews are very common, as are moderated and un-moderated user *testing* sessions—where a version of a UI is shared with many users in a structured fashion. 
+
```

Comment:
I do think a common term that is missing from here that crops up in a lot of historical developer/OSS writing/literature (still mostly academic sadly) is the term 'Usability' and often this is the lens that OSS/Developers understand 'design' by so e.g. 'How usable is my OSS?' and often this is default translated into 'How usable is my OSS (by me and people that look, act and think like me)' .

Including the term here might help OSS contributors/developers feel less intimidated by some of the more subject matter expertise design terms so after we merge this I'll like to add a PR to add this term :)

Reply:
No need for a lengthy explanation, including language that ppl outside of design can identify with, is important! 

It has also not been my own experience, that "usability" is in any way restricted to academia. It has been very in-use in industry, the 18yrs I've been doing this—and it's imho an important term to understand. As are the origins of UX—which are not in academia, but in industry. 

That said, in OSS I have also observed the trend of honoring knowledge (and to an extent, practitioners) from academia, more than knowledge from where it was developed—which was in industry. As someone who's been doing UX for +18yrs, after a career in traditional design, that's been pretty erasive and shitty to experience. Academia is financially un-tenable to most Americans (and many others in the world), as are advanced degrees—and a subtext I've often picked-up in SimplySecure doings, is a preference to overlook practice development from industry folks, in favor of honoring academic developments. 

While America is not the center of the universe (don't tell her that!), there is an entire community of anti-Capitalistic/anti-Colonist UX professionals between here and Europe, hungry to do the kind of work OSS needs UX contributors to work on—and upholding academia as a source-of-truth (or where the most intellectually curious, exist; or where the most ethics-oriented practitioners exist) as often as this community does, has been a big problem I'd like to see confronted. 

Sorry, a bit of a tangent. But, an important thing I'd like to see this community address.